middleware/kubernetes/README.md

```diff
@@ -121,6 +121,59 @@ kubernetes coredns.local {
 	# Each line consists of the name of the federation, and the domain.
 	federation myfed foo.example.com
 	
+	# autopath [ndots:NDOTS] [resolv:RESOLV-CONF-FILE] [onnxdomain:RESPONSE] ...
```

Comment:
We should drop these option prefixes IMHO, we don't use those anywhere else.
So just `autopath [NDOTS] [RESOLVE-CONF] [RESPOSE]`